### PR TITLE
Update default limits

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: api/v1/types_util.go [hunk 1]
```diff
@@ -434,8 +430,12 @@ func (spec *InfinispanContainerSpec) GetMemoryResources() (requests resource.Qua
 }
 
 func getRequestLimits(str string) (requests resource.Quantity, limits resource.Quantity, err error) {
-	parts := strings.Split(str, ":")
+	if str == "" {
+		err = fmt.Errorf("resource string cannot be empty")
```
Comment (Crumby) [inline api/v1/types_util.go:434]: Shouldn't this be handled by CRD definition?

Comment (ryanemerson) [reply api/v1/types_util.go:434]: A CPU value of `""` is valid for the CR as that's the default, i.e no limit. This is simply a sanity check to ensure that nowhere in the code do we accidentally pass `""`. Calls to `GetCpuResources()` should only be made if there's a non-empty value.

**Review by pruivo — APPROVED** [review]
LGTM!

**Review by rigazilla — APPROVED** [review]
lgtm

**Review by Crumby — APPROVED** [review]

**Pull request merged**